modules/social_features/social_landing_page/social_landing_page.module

```diff
@@ -62,10 +62,16 @@ function social_landing_page_theme() {
   $theme_templates['group__featured'] = [
     'base hook' => 'group',
   ];
+  $theme_templates['group__featured__sky'] = [
+    'base hook' => 'group',
+  ];
```

Comment:
We're creating a new template/theme couple here. We probably want to adjust the template suggestions for the `group__featured` template instead.

Reply:
Templates were moved to socialblue theme (the same as for content).